GroceryMarket/Animation.swift

```diff
@@ -17,17 +17,36 @@ class Animation : NSObject {
 
 @objcMembers
 class TabBarFormat : NSObject {
-//    override func viewDidLoad(){
-//        super.viewDidLoad()
-//
-//        self.ChangeRadiusOfTabBarItem()
-//    }
     
     func ChangeRadiusOfTabBarItem(_ tabBar: UITabBar)  {
```

Comment:
nit: start the function name with lowercase - `changeRadiusOfTabBarItem`, same for all other functions.

Reply:
nit: always have one space between `)` and `{`. Same for all other places.